dnd-bot/dnd_bot/dc/ui/views/view_character_creation.py

```diff
@@ -129,13 +129,13 @@ async def next(self, button: nextcord.ui.Button, interaction: nextcord.Interacti
         try:
             await HandlerAlignment.handle_next(self)
             await Messager.edit_last_user_message(user_id=self.user_id,
-                                                  embed=MessageTemplates.class_form_view_message_template(),
+                                                  embeds=[MessageTemplates.class_form_view_message_template()],
                                                   view=ViewClassForm(self.user_id, self.token))
         except DiscordDndBotException as e:
             # check for previous error messages
             error_data = MessageHolder.read_last_error_data(self.user_id)
             if error_data is None:
-                await Messager.send_dm_message(user_id=self.user_id, content=str(e), error=True)
+                await Messager.send_dm_error_message(user_id=self.user_id, content=str(e))
```

Comment:
sprawdzenie error_data is None jest już niepotrzebne bo jest to sprawdzane w metodzie delete_last_user_message z której korzysta metoda send_dm_error_message więc można to usunąć z tego miejsca i z innych miejsc w tym pliku